pkg/kubelet/eviction/helpers.go

```diff
@@ -61,6 +61,13 @@ var signalToResource = map[Signal]api.ResourceName{
 	SignalNodeFsAvailable:  resourceNodeFs,
 }
 
+// resourceToSignal maps a Resource to its associated Signal
+var resourceToSignal = map[api.ResourceName]Signal{
```

Comment:
Could you make a method to generate the based on signalToResource to avoid out-of-sync in the future? At least there should be comment about the 2 maps should be updated together.